detectable/src/test/java/com/synopsys/integration/detectable/functional/DetectableFunctionalTest.java

```diff
@@ -167,6 +167,14 @@ public void addExecutableOutput(@NotNull final ExecutableOutput executableOutput
         executableRunner.addExecutableOutput(executable, executableOutput);
```

Comment:
I would change DetectableFunctionalTest::addExecutableOutput to take in the working directory for the executable instead of creating two more methods that do essentially the same thing.

```
public void addExecutableOutput(@NotNull final ExecutableOutput executableOutput, @NotNull final Map<String, String> environment, @NotNull final String... command) {
    addExecutableOutput(sourceDirectory, executableOutput, environment, command);
}

public void addExecutableOutput(@NotNull final Path workingDirectory, @NotNull final ExecutableOutput executableOutput, @NotNull final Map<String, String> environment, @NotNull final String... command) {
    final List<String> commandList = Arrays.asList(command);
    final Executable executable = new Executable(workingDirectory.toFile(), environment, commandList);
    executableRunner.addExecutableOutput(executable, executableOutput);
}
```

Reply:
Love it

Reply:
to preserve the current api, how do you feel about having an addExecutableOutput that defaults the workingDirectory to sourceDirectory

Reply:
looks in order to do this we'd either have to change the access modifiers on DetectableFunctionalTest so you can reference the directories, or just pass a boolean or enum to indicate which directory you want to add to